src/lcmap_client/data/surface_reflectance.py

```diff
@@ -4,10 +4,11 @@
 import numpy.ma as ma
 
 from lcmap_client.data import url
+from lcmap_client.util import proj_point_to_tile_point, transform_matrix
 
 
 log = logging.getLogger(__name__)
-context = url.context + "/surface-reflectance/tiles"
+context = url.context + "/surface-reflectance/"
```

Comment:
Man, it's going to be nice when all these strings are unified in a single project ... f